salt/modules/deb_postgres.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+'''
+Module to provide Postgres compatibility to salt for debian family specific tools.
+
+'''
+
+# Import python libs
+from __future__ import absolute_import
+import logging
+import pipes
+
+# Import salt libs
+import salt.utils
+
+# Import 3rd-party libs
+import salt.ext.six as six
+
+
+log = logging.getLogger(__name__)
+
+__virtualname__ = 'postgres'
```

Comment:
@kiorky this was deliberate, the command line is then unified for postgres operations. But if this is not common pratice I'm can change that.